src/Netflix/Netflix.ts

```diff
@@ -0,0 +1,835 @@
+// lipsurf-plugins/src/Netflix/Netflix.ts
+
+/**
+ * Limitations and gotchas:
+ * - This plugin relies on Netflix's `window.netflix` publicized object.
+ *   This plugin MAY fails if netflix introduce a change to that `window.netflix` object
+ * - Watch command relies on videos on cache. It only queries the show netflix has loaded from
```

Comment:
this comment needs to be updated to reflect that it now falls back to using netflix search directly if not in cache